src/bijectors/truncated.jl

```diff
@@ -68,6 +68,27 @@ end
 
 with_logabsdet_jacobian(b::TruncatedBijector, x) = transform(b, x), logabsdetjac(b, x)
 
+# from stan https://github.com/stan-dev/math/blob/develop/stan/math/prim/constraint/lub_constrain.hpp etc
+function truncated_inv_logabsdetjac(x, a, b)
```

Comment:
Maybe follow the convention above and use `y` for the argument of functions of the inverse bijector such as `truncated_invlink`?
 ```suggestion
function truncated_inv_logabsdetjac(y, a, b)
```